focus_areas/Code_Changes.md

```diff
@@ -0,0 +1,56 @@
+# Code_Changes
+
+Changes to the source code during a certain period.
+
+## 1. Description
+
+These are changes to the source code during a certain period.
+For "change" we consider what developers consider an atomic change to their code.
+In other words, a change is some change to the source code which usually
+is accepted and merged as a whole, and if needed, reverted as a whole too.
+For example, in the case of git, each "change" corresponds to a "commit",
+or, to be more precice, "code change" corresponds to the part of a commit which
+touches files considered as source code.
+
+### Parameters
```

Comment:
Parameters and Aggregators appear to be most suitable for section 3. Sample Filter and Visualization.